src/providers/route.js

```diff
@@ -216,6 +217,18 @@ angular.module('bicker_router').provider('Route', function(ObjectHelper) {
           return flashStates;
         },
 
+        setCurrentBinding(viewName, binding) {
```

Comment:
Technically the binding name is sufficient for this purpose but I'd rather not have to refactor this if/when we find out we have another use case that requires more info. The only reason I can see against using the object instead of the name is memory consumption, and the binding definitions aren't that big, particularly when we're only storing the currently "rendered" binding for each currently rendered view (which is limited to what... 3 currently?)